app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/home/HomeScreenShimmerLoading.kt

```diff
@@ -40,11 +40,9 @@ fun HomeScreenShimmerLoading(modifier: Modifier) {
 
 @Suppress("DestructuringDeclarationWithTooManyEntries")
 @Composable
-fun ShimmerHomeScreen(brush: Brush, modifier: Modifier) {
+fun ShimmerHomeScreen(modifier: Modifier, brush: Brush) {
```

Comment:
Same comment re optional modifier 👀 
```suggestion
fun ShimmerHomeScreen(brush: Brush, modifier: Modifier = Modifier) {
```